### DISCUSSION TITLE
Immich and Caddy ( handle_path)

### DISCUSSION THREAD

**Discussion by dsavicIre** [immich-app/immich#7089] [Q&A]
HI,

Did anyone successfully set up Immich behind Caddy reverse proxy using handle_path?

Like ...

xxx.xxxx.xxx
{
  handle_path /immich* {

    reverse_proxy http://192.168.1.108:2283
  }

  tls {
         issuer acme {
                email xxx@xxx.com
       }
}

All I am getting is an empty web page, I imagine something needs to be configured in .env ,

Thanks,
Dejan

**Comment by bo0tzz** [comment]
Running immich on a path is not supported.

**Reply by sjayanna01** [reply]
Sorry but what does "running on a path" not supported mean in caddy's context? or is it a immich limitation?

**Comment by graphixillusion** [comment]
I'm interested to this feature too. Still not possible?
